charts/spire/README.md

```diff
@@ -32,6 +32,43 @@ spec:
         - --service-account-signing-key-file=/run/config/pki/sa.key
 ```
 
+## Usage
+
+To utilize Spire in your own workloads you should add the following to your workload:
+
+```diff
+ apiVersion: v1
+ kind: Pod
+ metadata:
+   name: my-app
+ spec:
+   containers:
+     - name: my-app
+       image: "my-app:latest"
+       imagePullPolicy: Always
++      env:
```

Comment:
The ENV bit is hard to describe. It has no effect on any app I know of. So could be confusing to users when it doesn't "work"

Reply:
Agree removed. This is specific to an app that uses this env var.
I also didn't use it in this example.

https://github.com/philips-labs/spiffe-vault/blob/main/charts/spiffe-vault/templates/spiffe-vault.yaml